gke-scripts/00-common-env.sh

```diff
@@ -27,7 +28,7 @@ export ACCOUNT_ADMIN_PORT="50056"
 export ACCOUNT_NEG_NAME="account-neg"
 export ACCOUNT_SERVICE_IMAGE="gcr.io/${PROJECT_ID}/psms-wallet-example:1.00"
 export ACCOUNT_BACKEND_SERVICE_NAME="account-backend-service"
-export ACCOUNT_SERVER_CMD="/build/install/wallet/bin/account-server"
+export ACCOUNT_SERVER_CMD="./account-server"
```

Comment:
The original one `/build/install/wallet/bin/account-server` is the Java command in the Java Docker image. We will need to figure out how to handle these differences between languages

Reply:
It would be nicer if all languages put their binaries in the same place.